modules/storage/storage.go

```diff
@@ -0,0 +1,287 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/zosv2/modules"
+	"github.com/threefoldtech/zosv2/modules/storage/filesystem"
+)
+
+const (
+	cacheTarget = "/var/cache"
```

Comment:
@muhamadazmy @delandtj should we keep this path ?

Reply:
I have no attachment to this path. Any other path would work. the question is, is there a reason to move away from this path?

Reply:
well I want to avoid to have stuff spread all over the place, and I wanted to try to keep each modules have a clean separation.  So we don't really allow modules to write anywhere in the filesystem, but instead just in one directory only.

I think we should document all the path/location used by everything we have, once we have a clear view we can decide where to put what.

Reply:
https://github.com/threefoldtech/zosv2/issues/79